app/src/test/java/com/simplemobiletools/calculator/MoneyCalculatorImplTest.kt

```diff
@@ -90,7 +112,68 @@ class MoneyCalculatorImplTest {
     }
 
     @Test
-    fun calculateTax(){
+    fun calculateTaxBritishColumbia(){
+        calc.addDigit(2)
```

Comment:
This should be refactored into a method since it's being done 10 times

Reply:
What should be refactored? Each test is testing for a different province tax rate